users/app.py

```diff
@@ -54,15 +54,47 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def upsert_user_in_db(user_object, users_collection):
+    """Updates or inserts the user object into user_collection.
+
+    Adds the is_oganizer field to user_object with default value False.
+
+    user_object should look like
+        {"user_id": foobar,
+         "name": Foo Bar}
+
+    Documents in users_collection should look like
+        {"user_id": foobar,
+         "name": Foo Bar,
+         "is_organizer": False}
+
+    Returns:
```

Comment:
Why not throw an AssertionError instead? Doesn't missing fields/etc mean that the object was malformed?

In some cases, it can be useful to return the created object if you have it on hand, since you've filled in additional fields on the object.

Reply:
Raising AssertionError done https://github.com/knative-portability/large-events/pull/8/commits/5b0030e9b87cac0f799e170a1ef0c68d074bb520

Returning created object done https://github.com/knative-portability/large-events/pull/8/commits/829984772e7d09ad55fb674aaf4234d2cb04dda2